### PR TITLE
Python Assignments

### PR DESCRIPTION
**Pull request by AkshataDudhane**

### CODE REVIEW

File: Assignment1.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+    #3) writing a function to return wine not sold to anyone born on the first Monday of the month.
+
+    def val_weekday(self):
+        weekday=datetime.strptime(self.birthday, '%m/%d/%Y') #converting weekday to a datetime object and matching the date format
+        return weekday.weekday()!=0 or weekday.day>7
+
+    #4) writing a function for email validation
+    def check_email(self):
+        pattern = r'\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,7}\b' #regular expression
```
Comment (bikashgupta-zinrelo) [inline Assignment1.py]: Please read regex

File: Assignment1.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            if (int(zip_str[i])+1==int(zip_str[i+1]) or int(zip_str[i+1])+1==int(zip_str[i])): #checking if the current character and next character if find return true else false
+                return False
+        return True
+
+
+    #3) writing a function to return wine not sold to anyone born on the first Monday of the month.
+
+    def val_weekday(self):
+        weekday=datetime.strptime(self.birthday, '%m/%d/%Y') #converting weekday to a datetime object and matching the date format
+        return weekday.weekday()!=0 or weekday.day>7
```
Comment (bikashgupta-zinrelo) [inline Assignment1.py:76]: Please add comments with examples

File: Assignment1.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+            return True
+        else:
+            return False
+
+    #21 years 
+    def calculateAge(self):
+        birth=datetime.strptime(self.birthday, '%m/%d/%Y')
+        today = date.today()
+        age = today.year - birth.year
+        if(today.month<birth.month or (today.month==birth.month and today.day<birth.day)):
```
Comment (bikashgupta-zinrelo) [inline Assignment1.py:93]: Add comments with examples

**Review by AkshataDudhane — COMMENTED** [review]
Changes are done accordingly and the comments are added as per the given suggestions.

**Review by bikashgupta-zinrelo — APPROVED** [review]

**Pull request merged**